<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IParticleCollectionSettingsForm</class>
 <widget class="QWidget" name="IParticleCollectionSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="material_group">
     <property name="title">
      <string> Material options</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_tracksUseBaseLightModel">
          <property name="toolTip">
           <string>Use a less advanced light model for the tracks. This might lead to more clearly visible tracks (but transparency will not be taken into account)</string>
          </property>
          <property name="text">
           <string>Use base lighting</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="VP1MaterialButton" name="pushButton_matButton">
          <property name="text">
           <string>Material</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="cuts" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_cuts_momentum">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Momentum cuts</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QGridLayout" name="_5">
             <property name="verticalSpacing">
              <number>4</number>
             </property>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_cut_minpt">
               <property name="text">
                <string>Min</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_cut_minpt_gev">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> GeV</string>
               </property>
               <property name="maximum">
                <double>999.990000000000009</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox_cut_maxpt">
               <property name="text">
                <string>Max</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_cut_maxpt_gev">
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="suffix">
                <string> GeV</string>
               </property>
               <property name="maximum">
                <double>9999.989999999999782</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_momtype">
             <item>
              <property name="text">
               <string>Pt</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>P</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>13</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="VP1EtaPhiCutWidget" name="etaPhiCutWidget" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_additional_widgets"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="close_widget">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_close">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VP1MaterialButton</class>
   <extends>QPushButton</extends>
   <header>VP1Base/VP1MaterialButton.h</header>
  </customwidget>
  <customwidget>
   <class>VP1EtaPhiCutWidget</class>
   <extends>QWidget</extends>
   <header>VP1Base/VP1EtaPhiCutWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
